src/Faithlife.Analyzers/UntilCanceledAnalyzer.cs

```diff
@@ -50,7 +51,7 @@ private static void AnalyzeSyntax(SyntaxNodeAnalysisContext context, INamedTypeS
 				return;
 
 			var method = context.SemanticModel.GetSymbolInfo(invocation.Expression).Symbol as IMethodSymbol;
-			if (method?.ContainingType != asyncEnumerableUtility)
+			if (!Equals(method?.ContainingType, asyncEnumerableUtility))
```

Comment:
These will get changed to `SymbolEqualityComparer.Default.Equals` in a future update.